pkg/cvo/status.go

```diff
@@ -354,6 +360,33 @@ func (optr *Operator) syncStatus(ctx context.Context, original, config *configv1
 	return err
 }
 
+func setTaintedCapabilitiesCondition(config *configv1.ClusterVersion, tainted []configv1.ClusterVersionCapability,
+	now metav1.Time) {
+
+	if len(tainted) > 0 {
+		message := "The following capabilities could not be disabled: "
+		caps := make([]string, len(tainted))
+		for i, c := range tainted {
+			caps[i] = string(c)
+		}
+		message = message + strings.Join([]string(caps), ", ")
+
+		resourcemerge.SetOperatorStatusCondition(&config.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+			Type:               TaintedCapabilities,
+			Status:             configv1.ConditionTrue,
+			Reason:             "Attempt to disable previousily enabled capability",
+			Message:            message,
+			LastTransitionTime: now,
+		})
+	} else {
+		resourcemerge.SetOperatorStatusCondition(&config.Status.Conditions, configv1.ClusterOperatorStatusCondition{
+			Type:               TaintedCapabilities,
+			Status:             configv1.ConditionFalse,
+			LastTransitionTime: now,
```

Comment:
can we get an all-is-well reason/message here, like:

```console
$ git --no-pager grep -A1 AsExpected
pkg/cvo/availableupdates.go:                            Reason:  "AsExpected",
pkg/cvo/availableupdates.go-                            Message: "The update is recommended, because none of the conditional update risks apply to this cluster.",
```

Maybe `AsExpected` and `All enabled capabilities are requested by the configured spec.`, or some such?